teuthology/openstack/__init__.py

```diff
@@ -692,11 +692,10 @@ def main(self):
         self.setup_logs()
         set_config_attr(self.args)
         log.debug('Teuthology config: %s' % self.config.openstack)
-        for keyfile in [self.args.key_filename,
-                        os.environ['HOME'] + '/.ssh/id_rsa',
-                        os.environ['HOME'] + '/.ssh/id_dsa',
-                        os.environ['HOME'] + '/.ssh/id_ecdsa']:
-            if (keyfile and os.path.isfile(keyfile)):
+        if not isinstance(self.args.key_filename, list):
+            self.args.key_filename = [self.args.key_filename]
```

Comment:
setting back args here is not a good style, it is better to use separate variable for key filename list, like:
```
keyfile_list = (lambda x: x if isinstance(x, list) else [x])(self.args.key_filename)
```

Reply:
done